components/pipeline-service/staging/stone-stg-rh01/deploy.yaml

```diff
@@ -2462,10 +2462,10 @@ spec:
         settings:
           application-name: Konflux Staging
           custom-console-name: Konflux Staging
-          custom-console-url: https://konflux-ui.apps.stone-stg-rh01.l2vh.p1.openshiftapps.com/
-          custom-console-url-pr-details: https://konflux-ui.apps.stone-stg-rh01.l2vh.p1.openshiftapps.com/ns/{{
+          custom-console-url: https://console.dev.redhat.com/application-pipeline
+          custom-console-url-pr-details: https://console.dev.redhat.com/application-pipeline/ns/{{
```

Comment:
This URL seems to be diverging from the other staging deployment. Do you know if that's intentional?

Reply:
I do not know that for sure. Sometimes people are only updating a single cluster when they want to test a change at smaller scale. Having said that, I'm not worried about this one as #6262 updates these again and BTW this is how we found out about the inconsistency.